controllers/pokemon_controller.go

```diff
@@ -0,0 +1,19 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/rromodev/academy-go-q32021/queries"
+
+	"github.com/gin-gonic/gin"
+)
+
+func GetPokemon(c *gin.Context) {
+	idp := c.Param("id")
+	id, _ := strconv.Atoi(idp)
+	fmt.Println(id)
+	pokemon := queries.GetPokemonById(id)
```

Comment:
It would be nice for this function to also return an error, that way if something fails you could return an appropriate status code to the user